support/ci/appveyor.ps1

```diff
@@ -179,11 +179,14 @@ if((Test-DocsOnlyChanged -eq $true) -and (Test-ReleaseBuild -eq $false)) {
                     Copy-Item "/hab/pkgs/core/hab/*/*/bin/*" (Split-Path $habExe -Parent) -Force
                 }
             }
-            if(!(Test-PullRequest) -and $env:hab_components -ne "launcher") {
-                $env:HAB_BLDR_CHANNEL = $channel
-                & $habExe pkg exec core/hab-bintray-publish publish-studio
-                $env:HAB_BLDR_CHANNEL = $null
-                if ($LASTEXITCODE -ne 0) {exit $LASTEXITCODE}
+            if(!(Test-PullRequest)) {
```

Comment:
this whole block should move to line 181 above the closing brace that is on that line now and change `$env:hab_components -eq "studio"` to `$component -eq "bintray-publish"`